backend/server/shared_routes.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2023, Yahoo Inc.
+// Licensed under the terms of the MIT. See LICENSE file in project root for terms.
+
+package server
+
+import (
+	"net/http"
+
+	"github.com/ashirt-ops/ashirt-server/backend/contentstore"
+	"github.com/ashirt-ops/ashirt-server/backend/database"
+	"github.com/ashirt-ops/ashirt-server/backend/dtos"
+	"github.com/ashirt-ops/ashirt-server/backend/server/middleware"
+	"github.com/ashirt-ops/ashirt-server/backend/services"
+	"github.com/go-chi/chi/v5"
+)
+
+func bindSharedRoutes(r chi.Router, db *database.Connection, contentStore contentstore.Store) {
+	route(r, "GET", "/checkconnection", jsonHandler(func(r *http.Request) (interface{}, error) {
```

Comment:
@jkennedyvz you [mentioned](https://github.com/ashirt-ops/ashirt-server/pull/892#discussion_r1274208048) adding checkconnection to web.go - I'm assuming we would still need this in the API routes as well (if someone chooses to keep them separate); if that's the case, then adding it to shared, should work, correct? 

Also, if this is added to web, how does that work? ie if you go to localhost:8080/checkconnection on the browser, there's no page so it just errors out. or would you be accessing it via web/checkconnection through something like postman?